lib/ansible/modules/apt_key.py

```diff
@@ -313,13 +313,16 @@ def import_key(module, keyring, keyserver, key_id):
 
     global lang_env
     if keyring:
-        cmd = "%s --keyring %s adv --no-tty --keyserver %s --recv %s" % (apt_key_bin, keyring, keyserver, key_id)
+        cmd = "%s --keyring %s adv --no-tty --keyserver %s" % (apt_key_bin, keyring, keyserver)
     else:
-        cmd = "%s adv --no-tty --keyserver %s --recv %s" % (apt_key_bin, keyserver, key_id)
+        cmd = "%s adv --no-tty --keyserver %s" % (apt_key_bin, keyserver)
 
     # check for proxy
     cmd = add_http_proxy(cmd)
 
+    # add recv argument as last one
+    cmd = "%s --recv %s" % (cmd, key_id)
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example. Also, add a test for this change.

Reply:
What do you mean by "add a test" ?

Reply:
Suggestions for a unit test: https://gist.github.com/mgedmin/1a8eaf34cddc07cd8a7c29f13a7b654c (untested sketch)

Reply:
I added @mgedmin as collaborator to my fork

Reply:
Thanks for the commit @mgedmin !